grails-app/conf/BuildConfig.groovy

```diff
@@ -22,12 +22,12 @@ grails.project.dependency.resolution = {
     repositories {
         mavenLocal()
         mavenRepo ("http://nexus.ala.org.au/content/groups/public/") {
-            updatePolicy 'always'
+            updatePolicy "always"
```

Comment:
groovy style guide recommends to use single quotes if dealing with string constants